system/settings/win32.xml

```diff
@@ -29,13 +20,6 @@
     </category>
   </section>
   <section id="system">
-    <category id="videoscreen">
-      <group id="1">
-        <setting id="videoscreen.fakefullscreen">
```

Comment:
this should stay here and be removed from settings. it's only used by Windows and OSX and will disappear from OSX when I get time for this. I think with DX11 it also will get obsolete for Win32.